docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -249,9 +249,18 @@ admission controller](https://git.k8s.io/kubernetes/plugin/pkg/admission/default
 
   * `node.alpha.kubernetes.io/unreachable`
   * `node.alpha.kubernetes.io/notReady`
+
+This ensures that DaemonSet pods are never evicted due to these problems,
+which matches the behavior when this feature is disabled.
+
+## Taint Nodes by Condition
+
+In Kuberentes 1.8 we added an alpha feature that makes NodeController create Taints matching Node Conditions and in the same time disables Condition check in scheduler. This assures that Conditions doesn't have influence on what's scheduled on the Node and that user
```

Comment:
I would say "In Kubernetes 1.8 we added an alpha feature that makes NodeController create taints corresponding to node conditions, and disables the condition check in the scheduler (instead the scheduler checks the taints). This assures that Conditions don't affect what's scheduled onto the node and the user can choose to ignore some of the node's problems (represented as Conditions) by adding appropriate pod tolerations."

BTW in the future please please please do not put each paragraph in a single line, it makes it hard to comment in Github.

Reply:
Done.